Sources/SwiftStylable/Classes/Style/Dimensions/DimensionCollection.swift

```diff
@@ -33,14 +33,15 @@ public class DimensionCollection {
             for (name, dimensionValue) in dimensionsEntries {
                 if let size = dimensionValue as? CGFloat {
                     if let dimensionHolder = self._dimensionHolders[name] {
+                        dimensionHolder.reference = nil
                         dimensionHolder.size = size
                     } else {
                         self._dimensionHolders[name] = DimensionHolder(size: size)
                     }
                     dimensionsEntries.removeValue(forKey: name)
                     numParsedDimensions += 1
                 } else if let parentString = dimensionValue as? String, let dimensionHolder = self._dimensionHolders[parentString] {
-                    self._dimensionHolders[name] = dimensionHolder
+                    self._dimensionHolders[name] = DimensionHolder(reference: dimensionHolder)
```

Comment:
Zelfde als bij color: moeten we hier checken of deze name al bestaat, en zo ja alleen de reference wijzigen?

Reply:
Zie comment bij color